sympy/stats/symbolic_probability.py

```diff
@@ -61,6 +61,7 @@ class Probability(Expr):
     """
     def __new__(cls, prob, condition=None, **kwargs):
         prob = _sympify(prob)
+        is_commutative = True
```

Comment:
@oscarbenjamin @NikhilSDate `is_commutative = True` was supposed to be added this way or this was needed to be done in some other way and I made a mistake ; ( ??

Reply:
Done